Sketch.java

```diff
@@ -1,18 +1,218 @@
 import processing.core.PApplet;
+import processing.core.PImage;
+
+/**
+* A program that creates a game where ther user controls a dog and has to avoid snowflakes by moving around them or clicking them.  The user loses if they touch 3 snowflakes.
+* @author: T.Chu
+* 
+*/
 
 public class Sketch extends PApplet {
 
+  // creates variable for the image
+  PImage img;
+
+  // arrays for the x and y coordinates of the snowflakes
+  float [] snowX = new float[42];
+  float [] snowY = new float [42];
+  boolean[] blnHideStatus = new boolean[42];
+  
+  // variable for snowflake size 
+  int snowDiameter = 10;
+  
   public void settings() {
+
     size(400, 400);
+
   }
 
+  boolean upPressed = false;
+  boolean downPressed = false;
+  boolean leftPressed = false;
+  boolean rightPressed = false;
+  boolean wPressed = false;
+  boolean aPressed = false;
+  boolean sPressed = false;
+  boolean dPressed = false;
+
+  // sets the amount of player lives
+  int playerLives = 3;
```

Comment:
global variables should always be placed at the beginning before the methods but after the class